spectrocloud/resource_cluster_eks_flatten_test.go

```diff
@@ -178,3 +179,65 @@ func TestFlattenClusterConfigsEKS(t *testing.T) {
 		})
 	}
 }
+
+func TestFlattenClusterConfigsEKSPrivateCIDRS(t *testing.T) {
```

Comment:
we also need a coverage for toEksCluster() part @SivaanandM

Reply:
we don’t have any default values for private_access_cidrs  and public_access_cidrs, its a list option

Reply:
I have covered the unit test for private cidr change and will raise a separate ticket for toEksCluster() unit test PLT-763